pkg/apis/deviceplugin/v1/sgxdeviceplugin_types.go

```diff
@@ -31,7 +31,7 @@ type SgxDevicePluginSpec struct {
 	// Image is a container image with SGX device plugin executable.
 	Image string `json:"image,omitempty"`
 
-	// InitImage is a container image with tools (e.g., SGX NFD source hook) installed on each node.
+	// Recommendation is to leave this unset and prefer the SGX NodeFeatureRule.
```

Comment:
we could still keep the original description and just add the recommendation?
```suggestion
        // InitImage is a container image with tools (i.e., SGX NFD source hook) installed on each node
	// Recommendation is to leave this unset and prefer the SGX NodeFeatureRule instead.
```